src/labs/ViewLabs.tsx

```diff
@@ -38,7 +39,15 @@ const ViewLabs = () => {
 
   useEffect(() => {
     const fetch = async () => {
-      const fetchedLabs = await LabRepository.findAll()
+      const sortRequest: SortRequest = {
```

Comment:
A test should be added to cover this scenario. 

In this case, I think it is sufficient to test that the `findAll` method was called with the correct sort request.

Reply:
I have added the test.  Please let me know if any change is needed.